packit/utils.py

```diff
@@ -250,3 +254,39 @@ def get_namespace(url: str) -> str:
         return namespace
     except IndexError:
         return None
+
+
+def get_wildcard_resolved_sync_files(list_to_sync: List) -> List:
+    logger.debug("Packit synced files %s", list_to_sync)
+    real_files = []
+    for sync in list_to_sync:
+        src = glob.glob(sync["src"])
+        real_files.append({"src": src, "dest": sync["dest"]})
+    logger.debug(f"Resolved synced file {real_files}")
+    return real_files
+
+
+def sync_files(
+    files_to_sync: list, src_working_dir: str, dest_working_dir: str
```

Comment:
Shouldn't it be a `List` from typing? not sure about this one

Reply:
It should be same if you do not need to specify the content type.

Reply:
I will change it to `List`.